support/approved-access.adoc

```diff
@@ -0,0 +1,20 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="approved-access"]
+= Approved Access
+include::_attributes/common-attributes.adoc[]
+ifdef::openshift-dedicated[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+endif::[]
+:context: approved-access
+
+toc::[]
+
+Red{nbsp}Hat Site Reliability Engineering (SRE) managing and proactively supporting ROSA Clusters will typically not require access to customer data as part of the normal operations. In the unlikely event should you need access to customer data, the *Approved Access* functionality provides an interface for customers to review and *approve* or *deny* access requests.
+
+Access requests to customer data on ROSA clusters and the corresponding cloud accounts can be created by Red{nbsp}Hat SRE either in response to a customer-initiated support ticket or in response to alerts received by a Red{nbsp}Hat SRE, as part of the standard incident response process.
+
+// Approved access
+
+include::modules/support-submitting-a-case-enable-approved-access.adoc[leveloffset=+1]
```

Comment:
@fmcdonal Should we add a section on "How am I informed of Access Requests?"

In that section we can explain:

- By default, the Cluster Owner will receive an email informing them of a new Access Request
  - If customers require additional users over/above the Cluster Owner to receive the email, they can add additional Cluster contacts using these steps: https://docs.openshift.com/rosa/observability/logging/sd-accessing-the-service-logs.html#adding-cluster-notification-contacts_sd-accessing-the-service-logs
- Pending Access Requests will be highlighted in the HCC via the clusters list or Access Requests tab on the cluster overview for the specific cluster

Reply:
applied